docs/src/content/components/drawer.mdx

```diff
@@ -189,37 +256,16 @@ const PinkBox = props => (
   version={props.version}
 />
 
+### Drawer
```

Comment:
I'd add the prop tables for the other subcomponents too.

Reply:
They don't take anything but whatever div can take and children?